### ISSUE TITLE
Handle "There are no migrations to run" error

### ISSUE THREAD

**Issue by rosesonfire** [balmasi/migrate-mongoose#30]
When I run migration and all migrations are already in Up status, an error is thrown. But since I need to run migration with an automated script, every time this error comes up, my script breaks. Is there any way to bypass this error?

**Comment by ronanamsterdam** [comment]
@rosesonfire  a quickie here:

```
....
return migrator.run('up')
                    .then(successfulMigration => {
                        resolve(successfulMigration);
                    })
                    .catch(err => {
                        if (err.toString() === "Error: There are no migrations to run") {
                            console.warn("There are no migrations to run");
                            resolve();
                        } else {
                            reject(err);
                        }
                    });
```

or same thing using async/await + try/catch.

But yes this definitely should NOT be and throw Error, but a warning

**Comment by rosesonfire** [comment]
Looks great! But I don't want to write up a js script for this. I just want to use the `migrate` CLI. Yes, this should be a warning, not an error. Or whether it should through an error or a warning should be configurable.

**Comment by rosesonfire** [comment]
Looks great! But I don't want to write up a js script for this. I just want to use the migrate CLI. Yes, this should be a warning, not an error. Or whether it should throw an error or a warning should be configurable.

**Comment by rosesonfire** [comment]
Looks great! But I don't want to write up a js script for this. I just want to use the `migrate` CLI. Yes, this should be a warning, not an error. Or whether it should throw an error or a warning should be configurable.

**Comment by ronanamsterdam** [comment]
@rosesonfire agreed. @balmasi will you take a look at the PR if I submit one? cuz I see there's bunch of pending PRs and I'm not sure if you're still willing to maintain the repo.

**Comment by balmasi** [comment]
Released in 4.0.0

I appreciate bringing this to my attention!
